be/src/exprs/str_to_map.cpp

```diff
@@ -36,14 +36,20 @@ StatusOr<ColumnPtr> StringFunctions::str_to_map(FunctionContext* context, const
 
     // split first
     Columns split_columns{columns[0], columns[1]};
-    RETURN_IF_ERROR(StringFunctions::split_prepare(context, FunctionContext::FRAGMENT_LOCAL));
     ASSIGN_OR_RETURN(auto splited, StringFunctions::split(context, split_columns));
```

Comment:
My question is if it can work well when the new BE work with old FE.

Reply:
Already test, it works fine